versioned_docs/version-21.0/index.md

```diff
@@ -6,7 +6,7 @@ slug: '/'
 
 OpCon (Operations Console Cross-Platform Scheduler) is an enterprise-wide, heterogeneous workflow automation and orchestration platform.
 
-The current release is **OpCon 21.0.3*.
+The current release is **OpCon 21.0.4*.
```

Comment:
Can we add a 2nd asterisk to the end?  Current deployed version has a mistake that we are continuing.